stdout-mq.js

```diff
@@ -230,8 +238,22 @@ if (!configOptions.spawnProcess) {
     through.obj(t.write.bind(t), t.close.bind(t)), (err) => {
       if (err) {
         console.log(err);
-        forwardSignal('SIGKILL');
-        process.exit(34);
+        /**
+         * Exit the main process
+         * @returns {void}
+         */
+        const processExit = () => process.exit(34);
+        const processExitTimeout = setTimeout(processExit, 60 * 1000);
+
+        child.off('exit', childOnExit);
```

Comment:
You don't need this, use flag https://nodejs.org/api/child_process.html#child_process_subprocess_killed